funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/result_queue_subscriber.py

```diff
@@ -16,10 +18,17 @@ class ResultQueueSubscriber(multiprocessing.Process):
 
     """
 
+    QUEUE_NAME = "results"
+    ROUTING_KEY = "*.results"
+    EXCHANGE_NAME = "results"
+    EXCHANGE_TYPE = "topic"
+
     def __init__(
         self,
-        pika_conn_params: pika.connection.Parameters,
+        *,
+        conn_params: pika.connection.Parameters,
         external_queue: multiprocessing.Queue,
+        kill_event: multiprocessing.Event,
     ):
```

Comment:
Similar question as earlier: ought we accordingly update the docstring?

Reply:
Again, a good catch, thank you for this. Fixed as part of the rebase.